### DISCUSSION TITLE
Advice on how to switch organization and scopes

### DISCUSSION THREAD

**Discussion by trevorgithub** [npm/feedback#998] [Registry]
I'm looking for advice on the recommended approach to switch an organization and scope from one account to another.

An ex-employee has reserved an organization and scope related to our company against their personal npmjs account. As that person has left the company, the company is interested in having that organization and scope be tied back to a company controlled npmjs account.

The ex-employee is very willing to relinquish the organization and scope.  But I'm not sure what the best approach to do this would be.  I didn't notice any guidance in the documentation, but I did stumble across this post:

https://github.com/npm/feedback/discussions/677

which suggests we could have problems transferring the organization and scope. Before we attempt to switch the organization and scope, I'd value any guidance on the best approach to handle this transfer.

**Comment by ljharb** [comment]
If it's an organization, they can easily add current employees as owners, and then remove themselves.

If it's their personal account scope, the only way would be for them to convert that scope (their user account) into an org, and do the same thing, but that's not a good idea if they have any *other* packages they manage

**Reply by trevorgithub** [reply]
Thanks for the super quick reply.  Let me check back with them for details.
